.github/workflows/stale.yml

```diff
@@ -1,7 +1,7 @@
 name: 'Close stale issues'
 on:
     schedule:
-        -   cron: '* */2 * * *'
```

Comment:
[“At every minute past every 2nd hour.”](https://crontab.guru/#*_*/2_*_*_*)
So it previously executed 12*60 times... totally unnecessary